recipes/jsonpickle/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "jsonpickle" %}
+{% set version = "0.9.5" %}
+{% set sha256 = "cc25dc79571d4ad7db59d05ddb7de0d76a8d598cf6136e1dbeaa9361ebcfe749" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - jsonpickle
+
+about:
```

Comment:
Can you please add `license_file: COPYING` and `dev_url: https://github.com/jsonpickle/jsonpickle`